fix: clear the textFields after saving on rssfeedhomepage

amitamrutiya2210 opened the pull request:
Fixes #198 

Describe the changes you have made in this PR -

- added controller.clear() method to clear the text textFields
- added two functions that handles the clearing of the fields in Feed tab and Rules tab
- added comments for the controller functionality

Screenshots of the changes (If any) -

https://user-images.githubusercontent.com/93595710/223111004-374b04ed-f7e8-408e-9d88-effcb66ef84e.mp4

https://user-images.githubusercontent.com/93595710/223111005-22637e34-160b-4327-952f-57c07a3cdf5f.mp4

Akshatji800 commented:
The Applicable Feed and Use as Base Path and Start on load don't get cleared up after saving a rule. Please fix that as well.

amitamrutiya2210 commented:
I fixed that issue

> @amitamrutiya2210 The Applicable Feed and Use as Base Path and Start on load don't get cleared up after saving a rule. Please fix that as well.

I have fixed that issue, so now you can proceed with the review.
